src/BloomExe/web/ServerBase.cs

```diff
@@ -17,6 +18,8 @@ namespace Bloom.web
 {
 	public abstract class ServerBase : IDisposable
 	{
+		internal const string SlashBloomSlash = "/bloom/";
```

Comment:
Problem is, I don't really know why it was done. All our localhost: urls include it so we need to strip it off...possibly this constant should be more widely used, I just needed it in two places so I defined a constant. I'll at least improve the name.